appservice/src/deploy/localGitDeploy.ts

```diff
@@ -62,6 +62,7 @@ export async function localGitDeploy(client: SiteClient, fsPath: string, context
                         await tryPushAndWaitForDeploymentToComplete(true);
                     } else {
                         context.telemetry.properties.cancelStep = 'forcePush';
+                        context.errorHandling.suppressDisplay = true;
                         throw new UserCancelledError();
```

Comment:
I thought we automatically suppressed display for any UserCancelledError…?

Reply:
Yeah Stephen's right - I think the bug is here: https://github.com/microsoft/vscode-azuretools/blob/68d1dc9591575be239d9f0fecbada370408b4a98/appservice/src/utils/callWithMaskHandling.ts#L13

Reply:
What would be the actual solution here?  I was thinking about calling callWithTelemetryandError around callWithMaskHandling, but it'd duplicate the event in telemetry.

Reply:
Inside of `callWithMaskHandling` you need to check if the error is a UserCancelledError and then just throw that original error instead of trying to mask the message.

The problem is you're converting a UserCancelledError into a normal Error, which we don't handle the same way.